### PR TITLE
Rebased version of CoreFX renames

### PR DESCRIPTION
**Pull request by eanova**
This if my first rebase, hopefully it went right.  This should resolve the issues with library test builds failing.

@danmosemsft @jkotas

### CODE REVIEW

File: src/libraries/System.Text.Json/tests/Serialization/TestClasses.SimpleTestClass.cs [hunk 1]
```diff
@@ -122,7 +122,7 @@ public class SimpleTestClass : ITestClass
                 @"""MyDateTime"" : ""2019-01-30T12:01:02.0000000Z""," +
                 @"""MyDateTimeOffset"" : ""2019-01-30T12:01:02.0000000+01:00""," +
                 @"""MyGuid"" : ""1B33498A-7B7D-4DDA-9C13-F6AA4AB449A6""," +
-                @"""MyUri"" : ""https://github.com/dotnet/corefx""," +
```
Comment (jkotas) [inline src/libraries/System.Text.Json/tests/Serialization/TestClasses.SimpleTestClass.cs]: These changes are causing failures in the System.Text.Json tests

Comment (jkotas) [reply src/libraries/System.Text.Json/tests/Serialization/TestClasses.SimpleTestClass.cs]:
```
Assert.Equal() Failure\r\nExpected: https://github.com/dotnet/corefx\r\nActual: https://github.com/dotnet/runtime
```

**Comment by danmosemsft** [comment]
Can you please mark this place with a comment noting it must not change
https://github.com/dotnet/runtime/issues/347#issuecomment-570706714

File: src/libraries/System.Data.Odbc/src/DatabaseSetupInstructions.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
 ## In Fedora 24 container:
 - `docker ps` shows _id of existing Fedora 24 container
```
Comment (Youssef1313) [inline src/libraries/System.Data.Odbc/src/DatabaseSetupInstructions.md]:
```suggestion
- `docker ps` shows \_id of existing Fedora 24 container
```

**Review by Youssef1313 — CHANGES_REQUESTED** [review]
Escape the underscore

File: src/libraries/System.Text.Json/tests/Serialization/TestClasses.SimpleTestClass.cs (hunk 1 above)
Comment (eanova) [reply src/libraries/System.Text.Json/tests/Serialization/TestClasses.SimpleTestClass.cs]: Thanks Jan,  I'll look into this.

**Comment by eanova** [comment]
@jkotas @danmosemsft 

Fixed the System.Text.Json.Tests.   As well I've added the note to the file to warn against updating.

**Comment by jkotas** [comment]
Could you please resolve the conflicts?

**Comment by eanova** [comment]
It says I need write access to resolved the conflicts.

**Comment by jkotas** [comment]
You can resolve them locally on your machine (e.g. by doing `git merge`) and then push the result to github.

**Review by jkotas — APPROVED** [review]
Thank you!

**Pull request merged**